rust/src/adapter/otel_formatter/mod.rs

```diff
@@ -177,16 +178,6 @@ impl Span {
     }
 }
 
-pub fn new_trace_id() -> String {
-    // This is how OpenTelemetry does it:
-    // https://docs.rs/opentelemetry_api/0.19.0/src/opentelemetry_api/trace/span_context.rs.html#137
-    format!("{:032x}", rand::thread_rng().gen::<u128>())
-}
-
-pub fn new_span_id() -> String {
-    format!("{:016x}", rand::thread_rng().gen::<u64>())
-}
-
```

Comment:
On the generic level we now pass around TelemetryId. The adapter can choose to conver this to hex encoded string or take the inner u64 type.

Reply:
OpenTelemetry wants the trace-id to be 16 bytes and the span-id to be 8 bytes https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/trace/v1/trace.proto#L87. It looks like TelemetryID can only support 8 bytes at the moment.

Reply:
okay, i think worth expanding it then

Reply:
Expanded to u128 and changed otel so it can use 32 and 16 chars